package.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Why create package.json for this?

Reply:
Edgar helped make this so I can just run 'npm run devSetup' and it'll link all the repos, it was in there the last version I accidentally removed it.

Reply:
lol, ok yeah that really comes handy, is this documented in the readme?

Reply:
yup it is